code/modules/vehicles/mecha/combat/neovgre.dm

```diff
@@ -1,68 +1,59 @@
-/obj/mecha/combat/neovgre
+/obj/vehicle/sealed/mecha/combat/neovgre
 	name = "Neovgre, the Anima Bulwark"
-	desc = "Nezbere's most powerful creation, a mighty war machine of unmatched power said to have ended wars in a single night."
+	desc = "Nezbere's most powerful creation, a mighty war machine of unmatched power said to have ended wars in a single night. Armed with a heavy laser and a tesla sphere generator. Requires a pilot and a gunner."
```

Comment:
Description appears inaccurate, it only has a laser and (as of now) does not appear to require a second occupant.

Reply:
I'll give a look into it.

Reply:
Second seat added.